src/citrine/resources/branch.py

```diff
@@ -160,3 +172,63 @@ def restore(self, uid: Union[UUID, str] = None):
         url = format_escaped_url(restore_path_template, project_id=self.project_id, branch_id=uid)
         data = self.session.put_resource(url, {}, version=self._api_version)
         return self.build(data)
+
+    def data_updates(self, uid: Union[UUID, str]) -> BranchDataUpdate:
+        """
+        Get data updates for a branch.
+
+        Parameters
+        ----------
+        uid:  Union[UUID, str]
+            Unique identifier of the branch to get data updates for
+
+        Returns
+        -------
+        BranchDataUpdate
+            A list of data updates and compatible predictors
+
+        """
+        path_template = '/projects/{project_id}/branches/{branch_id}/' + \
+                        'data-version-updates-predictor'
+        path = format_escaped_url(path_template, project_id=self.project_id, branch_id=uid)
+        data = self.session.get_resource(path, version=self._api_version)
+        return BranchDataUpdate.build(data)
+
+    def next_version(self,
```

Comment:
All parameters (except branch ID) should be keyword only. https://github.com/CitrineInformatics/citrine-python/blob/main/CONTRIBUTING.md#coding-style

Reply:
And since this specifically uses the branch root ID instead of the branch ID, it should probably also be a keyword argument.